pipeline/tests/asm.rs

```diff
@@ -731,3 +731,9 @@ fn connect_no_witgen() {
     let i = [];
     verify_asm(f, slice_to_vec(&i));
 }
+
+#[test]
+fn generics_preservation() {
+    let f = "asm/generics_preservation.asm";
+    verify_asm(f, Default::default());
```

Comment:
Is there anything else to take out of here? Or maybe I should move the test somewhere else?

Reply:
I think we don't really have good tests for assembly apart from the pipeline tests. You can see if you can find a better place also for future tests